contrib/init/systemd/README.md

```diff
@@ -3,20 +3,37 @@
 > NOTE: These instructions are valid for Ubuntu versions using `systemd` init
 > system.
 
-Run the following commands:
+## TL;DR
+1. Get the latest [release](https://github.com/Azure/azurefile-dockervolumedriver/releases)
+2. Put the binary into `/usr/bin/azurefile-dockervolumedriver`
+3. Get the .default and .service files from the .tar.gz file and deploy them
+4. Reload systemd
+
+## In-depth walkthrough
+To ssh onto machines created with the `docker-machine` commands, you'll probably need to ssh via `docker-machine ssh [machinename]` to perform the following.
 
 0. `sudo -s`
-0. Download the binary from “Releases” tab of the repo to `/opt/bin/`: `wget -qO/usr/bin/azurefile-dockervolumedriver [url]`
-0. Make it executable: `chmod +x /usr/bin/azurefile-dockervolumedriver`
+0. Download the tar.gz from "Releases" tab of the repo
+    + Currently this would be `wget https://github.com/Azure/azurefile-dockervolumedriver/archive/0.2.1.tar.gz` 
+0. Extract and copy the extracted files to `/usr/bin/azurefile-dockervolumedriver`: 
+    + Decompress the archive:  `tar -xvf 0.2.1.tar.gz`
+0. Download the binary from the "Releases" tab of the repo to `/usr/bin/azurefile-dockervolumedriver`
+    + Use wget: `wget -qO/usr/bin/azurefile-dockervolumedriver https://github.com/Azure/azurefile-dockervolumedriver/releases/download/0.2.1/azurefile-dockervolumedriver`
+    + Make it executable `chmod +x /opt/bin/azurefile-dockervolumedriver`
 0. Save the `.default` file to `/etc/default/azurefile-dockervolumedriver`
+    + Copy the file: `cp azurefile-dockervolumedriver-0.2.1/contrib/init/systemd/azurefile-dockervolumedriver.default /etc/default/azurefile-dockervolumedriver`
```

Comment:
I think it's important users know where to get the files - not least because they might end up with the wrong copies from the init folder. If you think that it's easier for them to get the file from the git dirs that's fine - although you'd still have an issue with restricted folder structures?